File: app/src/main/java/com/cristhian/gametobeat/ui/widget/GameWidgetService.java
package com.cristhian.gametobeat.ui.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class GameWidgetService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new GameRemoteViewsFactory(this.getApplicationContext());
    }
}

File: app/src/main/java/com/cristhian/gametobeat/ui/widget/GameWidgetService.kt
package com.cristhian.gametobeat.ui.widget

import android.content.Intent
import android.widget.RemoteViewsService

class GameWidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return GameRemoteViewsFactory(this.applicationContext)
    }
}